File: app/src/main/java/com/wlmtest/forecasttestjava/model/fivedaysforecastdata/FiveDaysForecastData.java

package com.wlmtest.forecasttestjava.model.fivedaysforecastdata;

import com.squareup.moshi.Json;

public class FiveDaysForecastData {

    @Json(name = "cod")
    private String cod;
    @Json(name = "message")
    private Double message;
    @Json(name = "cnt")
    private Integer cnt;
    @Json(name = "list")
    private java.util.List<List> list = null;
    @Json(name = "city")
    private City city;

    public String getCod() {
        return cod;
    }

    public void setCod(String cod) {
        this.cod = cod;
    }

    public Double getMessage() {
        return message;
    }

    public void setMessage(Double message) {
        this.message = message;
    }

    public Integer getCnt() {
        return cnt;
    }

    public void setCnt(Integer cnt) {
        this.cnt = cnt;
    }

    public java.util.List<List> getList() {
        return list;
    }

    public void setList(java.util.List<List> list) {
        this.list = list;
    }

    public City getCity() {
        return city;
    }

    public void setCity(City city) {
        this.city = city;
    }

}

File: app/src/main/java/com/wlmtest/forecasttestjava/model/api/fivedaysforecastdata/FiveDaysForecastData.kt
package com.wlmtest.forecasttestjava.model.api.fivedaysforecastdata

import com.squareup.moshi.Json

class FiveDaysForecastData {

    @Json(name = "cod")
    var cod: String? = null
    @Json(name = "message")
    var message: Double? = null
    @Json(name = "cnt")
    var cnt: Int? = null
    @Json(name = "list")
    var list: java.util.List<List>? = null
    @Json(name = "city")
    var city: City? = null

}
